src/gl/context.js

```diff
@@ -113,6 +114,8 @@ class Context {
         if (this.extTextureHalfFloat) {
             gl.getExtension('OES_texture_half_float_linear');
         }
+
+        this.extTimerQuery = gl.getExtension('EXT_disjoint_timer_query');
```

Comment:
@ansis, do we have a matrix of browsers where this is supported? Related to https://www.chromium.org/Home/chromium-security/ssca

Reply:
@ansis What is the current behavior on non-Chrome browsers? Do the event handlers ever get called? Are there any errors thrown?

Reply:
Yes, it is re-enabled after Chrome 70 on Desktop only: https://chromium.googlesource.com/chromium/src.git/+/cc1732e0386125c0c3aa04690663bfb43e6d7f68

I didn't see that it's precision is reduced, as it is done for performance.now (https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/timing/performance.cc?sq=package:chromium&dr=CSs&g=0&l=893).